src/libraries/System.Text.Json/src/System/Text/Json/Serialization/Converters/JsonValueConverterChar.cs

```diff
@@ -8,12 +8,17 @@ namespace System.Text.Json.Serialization.Converters
 {
     internal sealed class JsonConverterChar : JsonConverter<char>
     {
-        public override char Read(ref Utf8JsonReader reader, Type typeToConvert, JsonSerializerOptions options)
+        public override char Read(ref Utf8JsonReader reader, Type typeToConvert, JsonSerializerOptions? options)
```

Comment:
`options` shouldn't be nullable. Same with `Write`

Reply:
As i remember we wrote tests for null options https://github.com/dotnet/corefx/pull/39750, but that must be just for avoid NRE, just mentioning, will make them non null

Reply:
Yep. Having tests where the user pass in null, and we make sure to have checks is different from whether or not `null` is expected as part of the API contract. In this case, it isn't.